server/basket/router.ts

```diff
@@ -88,13 +110,60 @@ router.delete(
   }
 );
 
+/**
+ * Modify multiple baskets' information
+ *
+ * @name PATCH /api/baskets
+ *
+ * @param {string} name - The given name for the item
+ * @param {{value: number, unit: string}} quantity - The number of items and its unit
+ * @param {{new: string, baskets: Array<Object>}} baskets - The baskets
+ * @return {BasketResponse[]} - the updated baskets
+ * @throws {403} - if the user is not logged in
+ * @throws {400} - If the basket name is empty or a stream of empty spaces
+ * @throws {409} - If the basket name already exists
+ */
+ router.patch(
+  '/',
+  [
+    userValidator.isUserLoggedIn,
```

Comment:
i think we also need a middleware function to check that no duplicated ingredients are added ;-; (e.g. can't add an ingredient under a duplicated ingredient name)

Reply:
- I think the middleware to check basket names isn't necessary because all the existing baskets should have valid names and if a new one is created, that is checked on the frontend before submitting the form. 
- duplicate items taken care of

Reply:
ah i see, sounds good thank you!!